### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by rodehoed**
Make it more clear

### CODE REVIEW

File: plugins/foreman_openscap/0.4/index.md [hunk 1]
```diff
@@ -75,7 +75,12 @@ Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap)
 
 ### 2.1 Installing foreman_openscap
 
-Please refer to Foreman install plugin [manual](http://theforeman.org/manuals/1.8/index.html#6.1InstallaPlugin) to install the plugin on Foreman.
+On the Foreman server:
+
+	> yum install ruby193-rubygem-foreman_openscap.noarch
+	> service httpd reload
+
+Please refer the Foreman plugin [manual](http://theforeman.org/manuals/1.8/index.html#6.1InstallaPlugin) for more information about installing Foreman plugins.
```
Comment (shlomizadok) [inline plugins/foreman_openscap/0.4/index.md]: Maybe update to 1.9 ? (`http://theforeman.org/manuals/1.9 ....`)

File: plugins/foreman_openscap/0.4/index.md [hunk 2]
```diff
@@ -75,7 +75,12 @@ Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap)
 
 ### 2.1 Installing foreman_openscap
 
-Please refer to Foreman install plugin [manual](http://theforeman.org/manuals/1.8/index.html#6.1InstallaPlugin) to install the plugin on Foreman.
+On the Foreman server:
+
+	> yum install ruby193-rubygem-foreman_openscap.noarch
```
Comment (shlomizadok) [inline plugins/foreman_openscap/0.4/index.md]: should be `yum install ruby193-rubygem-foreman_openscap`

File: plugins/foreman_openscap/0.4/index.md (hunk 1 above)
Comment (domcleal) [inline plugins/foreman_openscap/0.4/index.md]: "latest" may be best

**Comment by mmoll** [comment]
merged, thanks @rodehoed!

**Pull request closed without merging**